src/main/java/org/spongepowered/common/data/provider/inventory/InventoryDataProviders.java

```diff
@@ -25,16 +25,15 @@
 package org.spongepowered.common.data.provider.inventory;
 
 import org.spongepowered.common.data.provider.DataProviderRegistratorBuilder;
-import org.spongepowered.common.data.provider.DataProviderRegistry;
 
 public final class InventoryDataProviders extends DataProviderRegistratorBuilder {
 
-    public InventoryDataProviders(final DataProviderRegistry registry) {
-        super(registry);
+    public InventoryDataProviders() {
+        super("inventory");
```

Comment:
Having all of these string constants, I almost want to refer to them through `o.s.c.util.Constants`